src/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--31">
+      <h1 class="calendar__title">Calendar</h1>
+      <div class="calendar__content">
```

Comment:
```suggestion
      <h1 class="calendar__title">Calendar</h1>
      
      <div class="calendar__content">
```

Separate block elements with a blank line for better code readability. Check other cases by yourself.

Reply:
Ok, good advice